### ISSUE TITLE
HRT Design Prototype in UX Pin (Intro page/Form) 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#29101]
## Issue Description
_ORDMI PIFs have never actually produced any design mockups for the harassment tool. They took conversations with business owners, synthesized requirements from these, then directly put them into code using the https://VA.gov CMS, strictly to save time. They will share screenshots the prototype and the business requirements assembled.  We'll need to take these screenshots and requirements and produce envision mockups that are compliant to VSA/VA.gov design systems.  The designs will be used for both design intent review and initial FE build of the prototype_

[Draft Design Sketch Review](https://www.sketch.com/s/b3f90417-a2a0-41f5-8fb9-151ce163710a)

[Link to Design Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26765) 
## Design tasks
- Create a prototype in UX pin for user research testing (Intro page/Form)
- Test prototype to ensure functionality is working as intended
- Validate with @cindymerrill ISO research efforts 
- Update/Finalize as needed 

## Acceptance Criteria
- [ ] Designer comments on this issue to approve prototype for review
- [ ] Designer posts link to Prototype for review by PIF/PM/PO
- [ ] Researcher reviews prototype ISO research plan
- [ ] Feedback is documented on ticket for follow up and review

**Comment by brianalloyd** [comment]
Created a new ticket for prototype build in UX pin.  Let's review timing for this ticket in planning.

**Comment by callen2563** [comment]
I have created the first draft [UX Pin prototype](https://preview.uxpin.com/2421e73be8eba75acea37ad3fbdd09bfc147940f#/pages/141512978?mode=chidm) 
However, once we decide what the testing plan is we can change the interactions as needed. Also, the copy is the first draft I was given so there are areas that are still FPO and will be updated once Danielle is finished with the landing page copy.

**Comment by cindymerrill** [comment]
I would like to take a look. If I have feedback, do you want them here in this ticket, @callen2563? Or in [#27438 [Discovery] Harassment Reporting Tool - Content Copy Review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27438)  if content-related?

**Comment by cindymerrill** [comment]
Fun to navigate through the prototype, @callen2563! I'll wait to give feedback until after more of the content is updated.

**Comment by callen2563** [comment]
@cindymerrill Yes, any comments for the prototype should go on this ticket.

**Comment by brianalloyd** [comment]
@callen2563 great work on the prototype build, thinking we should close out this ticket and create a "second draft" ticket in support of content update and finalization.  Thoughts?

**Comment by brianalloyd** [comment]
@callen2563 going to close this out for this sprints work and opened up ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/29718 for finalization of prototype build this coming sprint.  Let's push any comments to the new ticket in support.
